pkgs/development/python-modules/tiledb/default.nix

```diff
@@ -0,0 +1,26 @@
+{  pkgs ? import ./pkgs.nix  }:
+with pkgs;
+let
+  python = python36;
+ 
+  tiledb  = import ./tiledb.nix { inherit pkgs; };
+
+in
+  python.pkgs.buildPythonPackage rec {
+    inherit python;
+
+    pname = "tiledb";
+    version = "0.4.2";
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
Usually nixpkgs mainline prefers you fetch it from PyPi. See how this is done here: https://github.com/NixOS/nixpkgs/pull/54404/files

Reply:
ok